docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support the throttle parsing and modes described in this section of the documentation, documentation for servo module parsing will be provided in a
+separate section.
+
+.. table:: Servo Module Support for Throttle Commands and Modes
+
+	+-------------+------------------------------------+
+	| Module      | Throttle Support                   |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                |
+	+-------------+------------------------------------+
+
+.. _throttle_def:
+
+Throttle Definition
+==========================
+This section defines what is meant by a “throttle command” in the rest of this document. **A throttle command is a command that can come from a variety of different sources 
+that encodes a value from 100% to -100%. The magnitude of this percentage controls how quickly the module should spin, and the sign indicates the direction of the spin.**
+
+The exact drive voltage/speed and direction that a throttle command commands depends on how the module is configured by the user. A throttle command of 100% will command 
+the module to spin at 100% of whatever the maximum set by its mode is, and in the direction set by its direction configuration. 
+A throttle command of -100% will command the module to spin at 100% of whatever the maximum set by its mode is in the opposite direction of its direction configuration. 
+The details of these configurations are covered in the :ref:`throttle_mode_maximums_directions` section. 
+
+.. _throttle_sources:
+
+Throttle Sources
+==========================
+There are currently three different types of throttle sources for a Vertiq module: IQUART, DroneCAN, and hobby protocols. These sources correspond to different 
+communication protocols that can send throttle commands to a Vertiq module. Each source uses different messages and mappings to send its throttle commands, 
+as covered in the sections below. 
+
+The Vertiq module identifies and tracks what source an incoming throttle command comes from, which is relevant for the :ref:`armed_throttle_source_lockout` feature.
+
+.. _throttle_iquart_ref:
+
+IQUART
+*******
+The `raw_value <https://iqmotion.readthedocs.io/en/latest/modules/vertiq_8108_150.html#id15>`_ entry of the ESC Propeller Input Parser client can be used to send a throttle command over IQUART. 

```

Comment:
The ESC Propeller Input Parser client's <raw value link> entry can send throttle commands over IQUART